### PR TITLE
Contributing Translations

### PR DESCRIPTION
**Pull request by audstav**
@seanlip @DubeySandeep PTAL

Changes updated on: 
https://oppia-user-guide.readthedocs.io/en/latest/index.html

### CODE REVIEW

**Comment by seanlip** [comment]
Thanks, @audstav! Deferring review to @DubeySandeep since he is in charge of the contributor dashboard.

File: docs/contributor/contribute.rst [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Contributing to a lesson
+=========================
+
+Thank you for improving Oppia's lessons with your contributions! This section will cover how you can contribute:
```
Comment (DubeySandeep) [inline docs/contributor/contribute.rst]: Should we assume that they are interested in contribution OR that have already done the contribution?

If the first one then I think we should go with "Thanks for your interest in contributions in Oppia lessons." (or something around that maybe?

File: docs/contributor/contribute.rst [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Contributing to a lesson
```
Comment (DubeySandeep) [inline docs/contributor/contribute.rst:1]: Contributions can be in the form of questions i.e, not a part of the lesson (or so?). I'm not sure about the terminology here, @seanlip do you have any suggestions? Should we use a more general term here?

**Review by DubeySandeep — APPROVED** [review]
@audstav, This looks awesome! :) 

Should we add helpful tips for translators as well? (I'll use the link in the email if their suggestion gets rejected.)

File: docs/contributor/contribute.rst (hunk 2 above)
Comment (seanlip) [reply docs/contributor/contribute.rst:1]: Hm, I can't think of a better term at the moment -- and, in a way, topics can be broadly construed as lessons. Let's leave it as this for now and maybe we can change it later when we add more examples!

File: docs/contributor/contribute.rst (hunk 1 above)
Comment (seanlip) [reply docs/contributor/contribute.rst]: I agree with @DubeySandeep here -- I think we should go with something like what he suggests and that we should address people who are interested in contribution.

Comment (audstav) [reply docs/contributor/contribute.rst]: Yes good point, I changed it now. Done

**Comment by audstav** [comment]
@DubeySandeep I added a 'Guiding Principles' section PTAL. If there's anything else I've missed, let me know!

File: docs/contributor/contribute.rst (hunk 2 above)
Comment (audstav) [reply docs/contributor/contribute.rst:1]: I see your point... something more general like 'Contributing to Oppia' might be too vague. I'm wondering if we need to be _that_ technically accurate in the title if only one of the contributions doesn't directly pertain to a lesson?

**Review by seanlip — COMMENTED** [review]
My apologies -- I reviewed this a few days ago but somehow forgot to publish my comments. Doing so now.

File: docs/contributor/contribute.rst (hunk 2 above)
Comment (seanlip) [reply docs/contributor/contribute.rst:1]: For my part I'm happy to leave this to @audstav's discretion. (See comment I posted above in this thread, as well.)

**Review by DubeySandeep — APPROVED** [review]
LGTM!

**Pull request merged**